com.jetbrains.WebStorm.yaml

```diff
@@ -0,0 +1,63 @@
+app-id: com.jetbrains.WebStorm
+runtime: org.freedesktop.Sdk
+runtime-version: "18.08"
+sdk: org.freedesktop.Sdk
+command: webstorm-desktop
+separate-locales: false
+tags:
+  - proprietary
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --filesystem=host
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --filesystem=xdg-run/keyring
+  - --device=dri
+modules:
```

Comment:
You might want to add `gtk2` as a _shared module_. Markdown, AsciiDoc, and possibly other plug-ins, need GTK+ `2.x` in order to behave correctly.